mesos_slave/tests/test_check.py

```diff
@@ -1,49 +1,53 @@
 # (C) Datadog, Inc. 2018
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
-# (C) Datadog, Inc. 2018
-# All rights reserved
-# Licensed under a 3-clause BSD style license (see LICENSE)
-
-import json
-import os
-
 import pytest
 from six import iteritems
 
-from datadog_checks.mesos_slave import MesosSlave
-
-CHECK_NAME = 'mesos_master'
-FIXTURE_DIR = os.path.join(os.path.dirname(__file__), 'fixtures')
-
-
-def read_fixture(name):
-    with open(os.path.join(FIXTURE_DIR, name)) as f:
-        return f.read()
-
-
-@pytest.fixture
-def check():
-    check = MesosSlave(CHECK_NAME, {}, {})
-    check._get_stats = lambda v, x, y, z: json.loads(read_fixture('stats.json'))
-    check._get_state = lambda v, x, y, z: json.loads(read_fixture('state.json'))
-
-    return check
-
 
-def test_check(check, instance, aggregator):
-    check.check(instance)
+# TODO: Fix docker-compose.yml networking set up
```

Comment:
I'm fine with keeping that for later. I guess there was no easy way to fix the compose file?

Reply:
Yeah, I'll make a card 🙁